openapi/components/schemas/KycDocument/Settings/KycSettingsIdentity.yaml

```
@@ -1,85 +1,96 @@
       isIdentityDocument:
         type: integer
         minimum: 0
         maximum: 100
-        description: Weight of isIdentityDocument.
+        description: Weight added if the document does not resemble an ID.
       isPublishedOnline:
         type: integer
         minimum: 0
         maximum: 100
-        description: Weight of isPublishedOnline.
+        description: Weight added if an exact match of the document is not found online.
       firstName:
         type: integer
         minimum: 0
         maximum: 100
-        description: Weight of firstName.
+        description: Weight added if the customer's first name is not matched.
       lastName:
         type: integer
         minimum: 0
         maximum: 100
-        description: Weight of lastName.
+        description: Weight added if the customer's last name is not matched.
       expirationDate:
         type: integer
         minimum: 0
         maximum: 100
-        description: Weight of `expirationDate`.
+        description: Weight added if an expiration date is not detected on the document.
       dateOfBirth:
         type: integer
         minimum: 0
         maximum: 100
-        description: Weight of dateOfBirth.
+        description: Weight added if a date of birth is not detected on the document.
       matchesDateOfBirth:
         type: integer
         minimum: 0
         maximum: 100
-        description: Weight of matchesDateOfBirth.
+        description: Weight added if an identity document does not have a matching date of birth.
       issueDate:
         type: integer
         minimum: 0
         maximum: 100
-        description: Weight of issueDate.
+        description: Weight added if an issue date is not detected on the document.
       hasMinimalAge:
         type: integer
         minimum: 0
         maximum: 100
-        description: Weight of hasMinimalAge.
+        description: |-
+          Weight added if the document does not verify the minimal age limit.
+          Minimal age is 21+ the for USA and 18+ for all other countries.
       hasMatchingFaceProof:
         type: integer
         minimum: 0
         maximum: 100
-        description: Weight of hasMatchingFaceProof.
+        description: Weight added if an identity document does not have matching face proof.
       nationality:
         type: integer
         minimum: 0
         maximum: 100
-        description: Weight of nationality.
+        description: Weight added if a nationality is not detected on the document.
       documentSubtype:
         type: integer
         minimum: 0
         maximum: 100
-        description: Weight of documentSubtype.
+        description: |-
+          Weight added if the document is not one of the recognized document subtypes.
+          For a list of available subtypes, see [KycDocumentSubtypes](https://github.com/Rebilly/api-definitions/blob/main/openapi/components/schemas/KycDocument/KycDocumentSubtypes.yaml).
```

```suggestion
        description: Weight added if the document is not one of the recognized document subtypes.
```

I think we don't need link here.